lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -7,6 +7,7 @@ const { ServerlessError } = require('../../../../../../classes/Error');
 
 const originLimits = { maxTimeout: 30, maxMemorySize: 3008 };
 const viewerLimits = { maxTimeout: 5, maxMemorySize: 128 };
+const supportedRuntimes = ['nodejs6.10', 'nodejs8.10', 'nodejs10.x', 'python3.7'];
```

Comment:
Is `nodejs12.x` not yet supported?

Hardcoding runtimes implies a maintenance issue (we need to introduce code changes on every change in that).
Can we leave it for AWS validation, or maybe there's some way to confirm on that by issuing an AWS request?

Reply:
Node12 is supported in normal lambdas but tts not supported as a Lambda@Edge

Reply:
Ok - I removed this validation and move this information to the documentation.